app/assets/javascripts/vote.js

```diff
@@ -1,13 +1,13 @@
-function changeVoteCount(evt, data, status, xhr) {
+function changeVoteCount(evt, data, status, xhr, voteable_type) {
     response = evt['detail'][0];
-    element = $("#vote-count-" + response['id']);
+    element = $("#vote-" + voteable_type + "-count-" + response['id']);
```

Comment:
If you have ES6 transpiling, then you could use the backticks with variable interpolation to help clean this up.

eg, 
```js
element = $(`#vote-${voteable_type}-count-${response['id']}`);
```